### ISSUE TITLE
Cannot build Duplicati.GUI.TrayIcon project with gtk-sharp 3.0.0.0

### ISSUE THREAD

**Issue by warwickmm** [duplicati/duplicati#3044]
I have recently had an issue building the `Duplicati.GUI.TrayIcon` project.  The compilation fails with the following error:
```
Error CS1703: An assembly `glib-sharp' with the same identity has already been imported. Consider removing one of the references (CS1703) (Duplicati.GUI.TrayIcon)
```
I suspect that something about my build environment has changed, since I can no longer build old revisions that I was able to successfully build before.  It appears that gtk-sharp 3.0.0.0 was recently added to my GAC, and if I downgrade the gtk-sharp and gdk-sharp assemblies to version 2.12.0.0, the build is successful (see [patch](https://github.com/duplicati/duplicati/compare/master...warwickmm:test/downgrade_gtk-sharp)).  

Would anyone have any insight as to what may be causing the issue with gtk-sharp 3.0.0.0?

This is on Arch Linux with the 4.15.3-2-ARCH kernel and Mono version 5.4.1.

**Comment by kenkendk** [comment]
The reason why those files are included that way, is really to allow building on Windows where the packages are usually not found. I try to keep the project "checkout-n-build" ready as much as possible.

I am thinking that it somehow notices that you have a newer version and chooses to load that instead.

Could you try to replace the section with something that has an explicit version, like:
```xml
    <Reference Include="gdk-sharp, Version=2.12.0.0, Culture=neutral, PublicKeyToken=35e10195dab3c99f">
      <HintPath>..\..\..\thirdparty\appindicator-sharp\gdk-sharp.dll</HintPath>
    </Reference>
    <Reference Include="gtk-sharp, Version=2.12.0.0, Culture=neutral, PublicKeyToken=35e10195dab3c99f">
      <HintPath>..\..\..\thirdparty\appindicator-sharp\gtk-sharp.dll</HintPath>
    </Reference>
```

**Comment by warwickmm** [comment]
Thanks @kenkendk, that works.  I'm not sure what the issue is with gtk-sharp 3.0, but perhaps specifying a specific version in the `.csproj` file is a good thing to do.

Unfortunately, I don't have a Windows box to test this on.  Does your suggestion retain the desired behavior on Windows?

**Comment by warwickmm** [comment]
This was fixed in revision 7656f4c717cbd374af4f93fc5030f9d66dc5c61b.  Thanks @kenkendk.
